src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import "./styles/variables";
+@import "./styles/header";
+@import "./styles/catalog";
+@import "./styles/card";
+@import "./styles/stars";
+
+* {
   margin: 0;
```

Comment:
Don't reset margin using * selector. Better specify the exact tags you want to reset